diagnostic.md

```diff
@@ -5,7 +5,8 @@ Place your responses inside the fenced code-blocks where indivated by comments.
 1.  Describe a reason why a join tables may be valuable.
 
   ```md
-    # < Your Response Here >
+Lets you avoid storing data in more than one place, and is semantically eaiser to understand
```

Comment:
Yup, also important to mention is that they are valuable when dealing with many-to-many relationships